src/mail/mjml/partial/course_starting_at.mjml

```diff
@@ -0,0 +1,7 @@
+<mj-section background-color="#F5f5f5">
+  <mj-column width="100%" padding="20px">
+    <mj-text align="center" color="#000000" font-size="16px" padding="4px 25px">
+      {{course.starting_at}} {{time_zone}}
```

Comment:
Do you plan to retrieve the nearest start date of course run ?

Reply:
Not sure at all, it's only a sample mail for this commit